pallets/investments/src/lib.rs

```diff
@@ -224,7 +224,7 @@ pub mod pallet {
 		StorageMap<_, Blake2_128Concat, T::InvestmentId, OrderId, ValueQuery>;
 
 	#[pallet::storage]
-	pub(crate) type InvestOrders<T: Config> = StorageDoubleMap<
+	pub type InvestOrders<T: Config> = StorageDoubleMap<
```

Comment:
This enables writing these storages from outside which is not desired. We should at least add a note to revert this change after removing the migration. 

Since we only read, it appears to me that it would be beneficial on using existing /adding `getters`.

Reply:
Opened an issue to take care of this and other things after the upgrade: rhttps://github.com/centrifuge/centrifuge-chain/issues/1567

We have full control of what code makes it's way to production so it's not that opening up this visibility puts us at danger that ill-intended users or ourselves corrupting those storages. Adding `getters` imo is just adding code that's not needed aside from this migration that's to be executed once and deleted after so I'd prefer to keep this as is.